Cybereason: add new integration (#680)
SEKOIA-IO/documentation: docs/xdr/features/collect/integrations/endpoint/cybereason_malop_activities.md

uuid: 0de050fb-3f56-4c7a-a9b6-76bf5298a617
name: Cybereason MalOp activities
type: intake

## Overview

Cybereason offers a set of Endpoint Detection and Response (EDR) solutions. Through the Cybereason platform, all suspicious operations will be gathered in MalOps, a multi-stage visualizations of device activities.

Please find below information available in MalOp activities:

- the list of affected machines
- the list of affected users
- all suspicious network connections
- all suspicious executions

{!_shared_content/operations_center/detection/generated/suggested_rules_0de050fb-3f56-4c7a-a9b6-76bf5298a617_do_not_edit_manually.md!}

{!_shared_content/operations_center/integrations/generated/cybereason-malop-cef_do_not_edit_manually.md!}

## Configure

> This setup guide shows how to forward all MalOp activities to SEKOIA.IO.

Reviewer: ```suggestion
This setup guide will guide you into forwarding all MalOp activities to SEKOIA.IO.
```

Author: Applied

### Create your intake

On SEKOIA.IO, go to the [Intakes page](https://app.sekoia.io/operations/intakes/new) and generate a new intake with the `Cybereason MalOp Activities` format.
Keep aside the intake key.

### Setup the Syslog collector

> Check the [Rsyslog Transport](../../../ingestion_methods/rsyslog/) documentation to install and set up the syslog collector.

Author: Applied
